embedded-workshop-book/src/dealing-with-registers.md

```diff
@@ -1,9 +1,11 @@
 # Dealing with Registers
 
-Open the `src/bin/events.rs` file.
-
 In this and the next section we'll look into RTIC's event handling features. To explore these features we'll use the action of connecting a USB cable to the DK's port J2 as the event we'd like to handle.
 
+✅ Open the `src/bin/events.rs` file.
+
+We'll read the code and explain, what it does. 
```

Comment:
nit: i *think* that comma doesn't belong there but then again I am not exactly a grammar pro